src/domain/event/eventHero/__tests__/EventHero.test.tsx

```diff
@@ -209,7 +209,7 @@ test('should have event dates when super event is not defined', () => {
   expect(screen.getByText(dateStr)).toBeInTheDocument();
 });
 
-test('should have super event dates when super event is defined', () => {
+test('should have super event dates and event dates when super event is defined', () => {
```

Comment:
`should show event dates and super event dates if super event is defined`

Reply:
fixed